system-design/app-interoperability/application-package-definition.md

```diff
@@ -17,98 +28,186 @@ The application package has the following folder structure:
 ```yaml
 /                            # REQUIRED top-level directory 
 └── margo.yaml               # REQUIRED application description file in YAML Format 
-└── resources                # OPTIONAL folder with e.g., container binaries 
-└── manifest                 # REQUIRED folder containing the application manifest 
-    ├── helm-chart           # OPTIONAL: Helm chart (or reference to it) goes in this folder  
-        ├── Chart.yaml 
-        ├── charts 
-        │   └── ... 
-        ├── templates 
-        │   └── ... 
-        └── values.yaml 
-    └── docker-compose       # OPTIONAL: Docker Compose file goes in this folder 
-        └── compose.yaml
+└── resources                # OPTIONAL folder with application catalog resources e.g., icon, license file, release notes 
 ```
 
 ## Application Description
 
-The `margo.yaml` file is the application description. The purpose of this file is to present the application on an [application registry](./workload-orch-to-app-reg-interaction.md) or marketplace from where an end user selects the application to hand it over to the Workload Orchestration Software, which configures it and makes it available on the “app repo” for installation on the edge device (see Section [Workload Orchestration Agent](./workload-orchestration-agent.md)).
-
-> **Note**
-> The format of the application description is *inspired by* the [Open Application Model](https://github.com/oam-dev/spec) (OAM), however, the format defined here is a *narrow subset*, which is sufficient for the goals of the Margo specification at this point.
+The `margo.yaml` file is the application description. The purpose of this file is to present the application on an application catalog or marketplace from where an end user selects the application to hand it over to the Workload Orchestration Software, which configures it and makes it available for installation on the edge device (see Section [Workload Orchestration Agent](./workload-orchestration-agent.md)).
 
```

Comment:
@phil-abb why did you remove the link to the "Application Registry" and call it now "application catalog"? 
We need to make sure this is consistent throughout the margo spec.

Reply:
This is the original sentence:

>The margo.yaml file is the application description. The purpose of this file is to present the application on an application registry or marketplace from where an end user selects the application to hand it over to the Workload Orchestration Software, which configures it and makes it available on the “app repo” for installation on the edge device (see Section Workload Orchestration Agent).

I changed it because the end user doesn't select anything from the application registry. The application registry is where the application vendor puts the margo.yaml file and nothing is presented from there. The end user will select it from either an "app catalog" or a marketplace.

Reply:
OK, makes sense.